packages/node/src/integrations/tracing/fastify.ts

```diff
@@ -79,7 +78,7 @@ export function setupFastifyErrorHandler(fastify: Fastify): void {
 
         // Taken from Otel Fastify instrumentation:
         // https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/plugins/node/opentelemetry-instrumentation-fastify/src/instrumentation.ts#L94-L96
-        const routeName = reqWithRouteInfo.routeOptions?.url || reqWithRouteInfo.routerPath;
```

Comment:
Hey, thanks for the PR! Could we do the same change as they did in OTEL:

```js
const routeName = reqWithRouteInfo.routeOptions ? reqWithRouteInfo.routeOptions.url : reqWithRouteInfo.routerPath;
```

Otherwise, this stops working on older fastify versions, I guess, which we do not want!

Reply:
Oh i see! So the `routerPath` is kept for backwards compatibility.